airflow/cli/commands/kubernetes_command.py

```diff
@@ -90,7 +90,23 @@ def cleanup_pods(args):
     print('Loading Kubernetes configuration')
     kube_client = get_kube_client()
     print(f'Listing pods in namespace {namespace}')
-    list_kwargs = {"namespace": namespace, "limit": 500}
+    airflow_pod_labels = [
+        'dag_id',
+        'task_id',
+        'execution_date',
+        'try_number',
+        'airflow_version',
+    ]
```

Comment:
We should probably not look for **all** of these labels:

We could probably simplify this to see if there is `kubernetes_pod_operator` or `kubernetes_executor` in the labels.

Let me think a bit more on this to see if we are missing a case -- what happens if a **Celery Worker**  pod hangs around

Reply:
Hi @kaxil , a gentle ping for this :)

I'm not sure how Celery Worker will be involved in this though. IMO we are focusing on "_pods started by Airflow_", not "_pods running Airflow (like worker node)_"

Reply:
Yea go for it, I couldn't think of any reason :)